### DISCUSSION TITLE
https://www.bleepingcomputer.com/news/security/google-chrome-adds-app-bound-encryption-to-block-infostealer-malware/

### DISCUSSION THREAD

**Discussion by ghost** [moonD4rk/HackBrowserData#376] [General]
https://www.bleepingcomputer.com/news/security/google-chrome-adds-app-bound-encryption-to-block-infostealer-malware/

what now? if this is fully implemented this repo wont work anymore

**Comment by ghost** [comment]
any progress or notes on this

**Reply by mymirai-nikki** [reply]
what now? learn how to code first kid. and make a pull request

**Reply by ghost** [reply]
yes found something for it. it properly works now. Thanks for your comment.
